src/sidebars/favourites/tearout-directive.js

```diff
@@ -160,6 +161,10 @@
                         function boundsChangingEvent() {
                             if (insideFavouritesPane()) {
                                 reorderFavourites();
+                                dragService.setOtherInstance(null);
```

Comment:
@owennw I've attempted a further abstraction on the inner state by adding a `destroy` function and adding a new function for `overThisInstance`.